apps/tailwind-components/pages/input/Ref.story.vue

```diff
@@ -4,13 +4,25 @@ const tableId = ref<string>("Pet");
 const labelTemplate = ref<string>("${name}");
 const value = ref([{ name: "spike" }]);
 const value2 = ref([{ name: "spike" }]);
+
+const refExampleError = ref<boolean>(false);
+
+function onBlur() {
+  if (!value.value.length) {
+    refExampleError.value = true;
```

Comment:
maybe better do a blur counter? now this story shows more than the component is actually able to do. That might be confusing to naive users of the component lib.

Reply:
we could add that to our standard test container if you like.